README.md

```diff
@@ -182,6 +182,39 @@ Finally, you're ready to use Fission!
   Hello, world!
 ```
 
+### Enable persistent function logs (Optional)
+
+Build fluentd docker image
+
+```
+  $ cd $GOPATH/src/github.com/fission/fission/logger/fluentd
+  $ ./build.sh
+```
+
+Create Kubernetes service and deployment
+
+```
+  $ cd $GOPATH/src/github.com/fission/fission/fission-logger.yaml
+  $ vim fission-logger.yaml # replace information in yaml file
+  $ kubectl apply -f fission-logger.yaml
+```
+
+Display function logs
+
+```
+  # Currently only influxDB is supported, use --help to see more options
+  $ export FISSION_LOGDB=http://$(kubectl --namespace fission get svc influxdb -o=jsonpath='{..clusterIP}'):8086
```

Comment:
I don't understand why clusterIP would work here?  The clusterIP of a service is usually not accessible from outside the cluster.